README.md

```diff
@@ -46,15 +15,13 @@ $ npm install
 $ npm test
 ```
 
-## GitBook
+## mdBook
 
-GitBook is built automatically via Travis, and changes pushed to the `gh-pages` branch. If you want to build and preview the GitBook locally, [install npm](https://www.npmjs.com/get-npm) and and run the following commands from the root of the repository:
+To generate these pages, mdBook is used and built automatically via Travis, and changes pushed to the `gh-pages` branch. If you want to build and preview the mdBook locally, [install Rust and cargo](https://www.rust-lang.org/learn/get-started) and then run the following commands from the root of the repository:
 
 ```
-$ npm install
-$ npm start
+$ cargo install --vers "^0.2" mdbook
```

Comment:
There's `0.3` now, can/should we use that?

Reply:
Definitely. Let me update and test that when I get to my laptop.

Reply:
Done.